notebooks/official/generative_ai/text_embedding_api_semantic_search_with_scann.ipynb

```diff
@@ -0,0 +1,1183 @@
+{
```

Comment:
<p>Add a period at the end of  <a href="https://ai.googleblog.com/2020/07/announcing-scann-efficient-vector.html" target="_blank">Google ScaNN: Efficient Vector Similarity Search</a> instead of a comma</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>